models/classes/Lists/Business/Domain/ValueCollectionSearchRequest.php

```diff
@@ -38,6 +38,9 @@ class ValueCollectionSearchRequest
     /** @var string[] */
     private $excluded = [];
 
+    /** @var int */
+    private $offset = 0;
```

Comment:
According to this comment https://github.com/oat-sa/tao-core/pull/3270/files#r771116688

```suggestion
    private $offset = null;
```